.env.default

```diff
@@ -10,6 +10,7 @@ PORT="8000"
 FIXTURE_PORT="9000"
 KARMA_PORT="8001"
 CORS_PORT="8002"
+WEBEX_CONVERGED_ORG_ID=930ad5d6-d315-4c09-a18a-bad630e48646
```

Comment:
Should put this in quotes:

WEBEX_CONVERGED_ORG_ID="930ad5d6-d315-4c09-a18a-bad630e48646"